.github/workflows/cd-terraform-mgmt-deploy.yml

```
@@ -1,4 +1,4 @@
-name: "CI - Update OPS Infrastructure"
+name: "CD - Update Management Infrastructure"
```

```suggestion
name: "CD - Update mgmt Infrastructure"
```

Recommend we stick to the name of the environment which is `mgmt` so that it's more obvious in the GitHub Actions UI.

---

Change has been incorporated.